### PR TITLE
Include Dutch I18n

### PR DESCRIPTION
**Pull request by johanb**
Added missing Dutch translations

### CODE REVIEW

File: config/locales/nl.yml [hunk 1]
```diff
@@ -37,6 +37,16 @@ nl:
           form:
             toggle_advanced_options: Klik voor toegang tot meta tag instellingen en menu opties
             published_at: Publicatiedatum
+            custom_url: Custom URL
```
Comment (parndt) [inline config/locales/nl.yml]: Is custom really custom in Dutch??

Comment (johanb) [inline config/locales/nl.yml]: Not really. We don't have a single word for it, and the Dutch language absorbs many English words. I considered a few translations but they only lead to more confusion and ambiguity.

Comment (parndt) [inline config/locales/nl.yml]: Good to know

**Comment by simi** [comment]
Dankjewel!

**Pull request merged**